src/agent/src/main.rs

```diff
@@ -34,6 +34,7 @@ use std::process::exit;
 use std::sync::Arc;
 use tracing::{instrument, span};
 
+mod ccw;
```

Comment:
Hi @Jakob-Naucke - please could you make this change s390-specific by using appropriate `target_arch` annotations. For example:

```suggestion
#[cfg(any(target_arch = "s390x"))]
mod ccw;
```